components/atoms/CartIllustration/cart-illustration.tsx

```diff
@@ -3,13 +3,9 @@ interface CartIllustrationProps {
 }
 const CartIllustration = ({ classNames }: CartIllustrationProps) => {
   return (
-    <div
-      className={`inline-flex max-w-[176px] gap-1 items-center px-1.5 w-44 h-12 rounded-md border ${
-        classNames ?? classNames
-      }`}
-    >
-      <div className="h-[34px] w-[34px] border rounded bg-light-slate-5"></div>
-      <div className="w-[80px]  gap-2  rounded">
+    <div className={`inline-flex gap-1 items-center px-1.5 w-44 h-12 rounded-md border ${classNames ?? classNames}`}>
+      <div className="h-9 w-9 border rounded bg-light-slate-5"></div>
+      <div className="w-20  gap-2  rounded">
```

Comment:
```suggestion
      <div className="w-20 gap-2 rounded">
```

Have you tried to run the format script ??

Reply:
@NsdHSO I did try, but it isn’t fixing it. I will do the fix manually.